spec/features/user_views_lists_spec.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails_helper'
+
+feature 'User views all lists' do
+  scenario 'successfully' do
+    create(:list, title: 'Best Sci Fi')
+    book = create(:book, cover_image_url: 'http://placehold.it/140x100')
```

Comment:
Could refactor into a transient to make test more readable: https://github.com/thoughtbot/factory_girl/blob/master/GETTING_STARTED.md#custom-construction

`create(:list, :with_books)`

screen hero discussion notes:

https://gist.github.com/evan-007/b1cf360173e7c8c7344e